src/services/notifications.rs

```diff
@@ -33,28 +33,36 @@ pub async fn post_registration(
 ) -> ApiResult<()> {
     let client = context.client();
 
-    for safe_registration in registration_request.safe_registrations.iter() {
+    for safe_registration in registration_request.safe_registrations.into_iter() {
         let info_provider = DefaultInfoProvider::new(&safe_registration.chain_id, &context);
         let url = core_uri!(info_provider, "/v1/notifications/devices/")?;
         let backend_request =
             build_backend_request(&registration_request.device_data, safe_registration);
-        client
+        let response = client
             .post(url.to_string())
             .json(&backend_request)
             .send()
             .await?;
+
+        if !response.status().is_success() {
```

Comment:
This will stop the registration in case on an error, right? I thought we ignore.errors and try to register on all services that are relevant.

Also is it possible to do these registrations in parallel with async await?

Reply:
we can do that, that was the old behaviour. 

Regarding your second question, I could add a stream implementation here, I guess. I will have a look